ci/lint/check_bazel_team_owner.py

```diff
@@ -29,7 +30,7 @@ def perform_check(raw_xml_string: str):
         for lst in rule.findall("list"):
             if lst.attrib["name"] != "tags":
                 continue
-            tags = [child.attrib["value"] for child in lst.getchildren()]
+            tags = [child.attrib["value"] for child in lst]
```

Comment:
no need `getchildren` any more?

Reply:
nope.. in fact, the function is not there anymore in python 3.9

I think the python version was python 3.8 in ubuntu 20.04.